Jenkinsfile

```diff
@@ -0,0 +1,54 @@
+pipeline {
```

Comment:
we should probably add some [options](https://jenkins.io/doc/book/pipeline/syntax/#options) to this for stuff like `buildDiscarder`